src/Symfony/Component/Form/Extension/HttpFoundation/Type/FormFlowTypeSessionDataStorageExtension.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Extension\HttpFoundation\Type;
+
+use Symfony\Component\Form\AbstractTypeExtension;
+use Symfony\Component\Form\Extension\Core\Type\FormFlowType;
+use Symfony\Component\Form\Flow\DataStorage\NullDataStorage;
+use Symfony\Component\Form\Flow\DataStorage\SessionDataStorage;
+use Symfony\Component\Form\Flow\FormFlowBuilderInterface;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+class FormFlowTypeSessionDataStorageExtension extends AbstractTypeExtension
+{
+    public function __construct(
+        private readonly ?RequestStack $requestStack = null,
+    ) {
+    }
+
+    /**
+     * @param FormFlowBuilderInterface $builder
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        if (null === $this->requestStack || !$builder->getDataStorage() instanceof NullDataStorage) {
+            return;
+        }
+
+        $key = \sprintf('_form_flow.%s_%s', strtolower(str_replace('\\', '_', $builder->getType()->getInnerType()::class)), $builder->getName());
```

Comment:
what about prefix with `_sf` like the http session attributes ?

Reply:
I think it's already the case, all Symfony session attributes are currently stored under `_sf2_attributes` key by default.